apps/condo/domains/user/schema/SupportMessagesForwardingService.test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Generated by `createservice user.SupportMessagesForwardingService`
+ */
+
+const { supportMessagesForwardingByTestClient } = require('@condo/domains/user/utils/testSchema')
+const { MESSAGE_SENDING_STATUS } = require('@condo/domains/notification/constants/constants')
+const { RU_LOCALE } = require('@condo/domains/common/constants/locale')
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+const conf = require('@core/config')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+const { makeLoggedInAdminClient, UploadingFile } = require('@core/keystone/test.utils')
+const { createTestResident } = require('@condo/domains/resident/utils/testSchema')
+const { addResidentAccess } = require('@condo/domains/user/utils/testSchema')
+const { registerNewOrganization } = require('@condo/domains/organization/utils/testSchema/Organization')
+const { createTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { Message } = require('@condo/domains/notification/utils/testSchema')
+const path = require('path')
+
+const EMAIL_API_CONFIG = (conf.EMAIL_API_CONFIG) ? JSON.parse(conf.EMAIL_API_CONFIG) : null
+const FORWARDING_EMAILS_FROM = 'doma-test-messages-forwarding@mailforspam.com'
+
+describe('SupportMessagesForwardingService', async () => {
+    test('Forward message to support: with attachments, with emailFrom', async () => {
+        const userClient = await makeClientWithProperty()
+        const adminClient = await makeLoggedInAdminClient()
```

Comment:
It's better don't use an admin client in these tests. You really don't need admin access to check your mutation

Reply:
I use `adminClient`, because I need to create a test resident in the beginning and check the message model at the end. Target mutation tested by `userClient`.